atomics/T1070.003/T1070.003.yaml

```diff
@@ -94,3 +94,43 @@ atomic_tests:
        hostname
        whoami
     name: sh
+- name:  Clear Powershell history (del (Get-PSReadlineOption).HistorySavePath)
+  auto_generated_guid: ''
+  description: |
+    Clears Powershell history
+  supported_platforms:
+  - windows
+  executor:
+    command: |
+       del (Get-PSReadlineOption).HistorySavePath
+    name: powershell
+- name:  Prevent Powershell history logging
+  auto_generated_guid: ''
+  description: |
+    Prvents Powershell history
+  supported_platforms:
+  - windows
+  executor:
+    command: |
+       Set-PSReadlineOption –HistorySaveStyle SaveNothing
+    name: powershell
+- name:  Clear Powershell history (Remove-Item (Get-PSReadlineOption).HistorySavePath)
+  auto_generated_guid: ''
+  description: |
+    Clears Powershell history
+  supported_platforms:
+  - windows
+  executor:
+    command: |
+       Remove-Item (Get-PSReadlineOption).HistorySavePath
```

Comment:
I suggest keeping only this version of the "Clear PowerShell History" atomic, since "del" and "rm" are simply aliases for "Remove-Item"

Reply:
Sounds good!

Reply:
Actually, if you haven't already made this change, go ahead and leave it here for now. There is some discussion among the maintainers as to where it should go but it could go either way at this point.